intg_test_manager.py

```diff
@@ -480,14 +480,14 @@ def build_module(module_path):
     """
     logger.info('Start building a module. Module: ' + str(module_path))
     if sys.platform.startswith('win'):
-        subprocess.call(['mvn', 'clean', 'install', '-fae', '-B',
+        subprocess.call(['mvn', 'clean', 'install','-P', 'testgrid','-fae', '-B',
```

Comment:
@piraveena Since the test-runner is a common component, this change might affect other products' jobs. We need to either use same profile across other products or provide a common approach that works for everyone.

WDYT?

Reply:
Already APIM did this with this PR https://github.com/wso2/product-apim/pull/5359